swan-accpy/Dockerfile

```diff
@@ -0,0 +1,37 @@
+ARG VERSION_PARENT=v0.0.18
+
+FROM gitlab-registry.cern.ch/swan/docker-images/jupyter/swan-cern:${VERSION_PARENT}
+
+LABEL maintainer="swan-admins@cern.ch"
+ARG NB_UID="1000"
+ARG BUILD_TAG=daily
+ENV VERSION_DOCKER_IMAGE=$BUILD_TAG
+ENV ACCPY_PATH="/opt/acc-py"
+
+RUN echo "Building swan-accpy image with tag ${VERSION_DOCKER_IMAGE} from parent tag ${VERSION_PARENT}."
+
+USER root
+
+RUN for version in "2020.11" "2021.12" "2023.06"; do \
+        wget http://bewww.cern.ch/ap/acc-py/installers/acc-py-${version}-installer.sh && \
+        chmod +x acc-py-${version}-installer.sh && \
+        ./acc-py-${version}-installer.sh --installation-root ${ACCPY_PATH} && \
+        rm -f acc-py-${version}-installer.sh && \
+        source ${ACCPY_PATH}/base/${version}/setup.sh && \
+        ${ACCPY_PATH}/base/${version}/bin/python -m ensurepip --upgrade && \
+        ${ACCPY_PATH}/base/${version}/bin/pip3 install ipykernel && \
+        deactivate; \
+    done
+
+RUN dnf install -y \
+    java-11-openjdk.x86_64 && \
+    dnf clean all && \
+    rm -rf /var/cache/dnf
+
+RUN pip install --no-deps --no-cache-dir \
+    swancustomenvironments==0.0.1
+
+# Jupyter notebook configuration - Add extension templates to jupyter
```

Comment:
Remove this comment, move the line below up right after the RUN above, with no newline in between them (they are both related to the extension).